hpack/src/main/java/com/twitter/hpack/ExtendedHeaderListener.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2015 Twitter, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.twitter.hpack;
+
+public interface ExtendedHeaderListener {
+
+  /**
+   * Called by the decoder during header field emission.
+   * The name and value byte arrays must not be modified.
+   *
+   * @param annotations array of length=2 which listeners can use to store parsed versions
+   *                    of the header and value and have HPACK cache them with indexed entries.
+   */
+  public void addHeader(byte[] name, byte[] value, Object[] annotations, boolean sensitive);
```

Comment:
let's just change the API for "HeaderListener" instead of adding this complexity.

bump the version in the README.md example pom dependency to 1.1.0 and modify the example to include the annotations array with some commented out code about caching a parsed representation

also change the version in pom.xml to 1.1.0-SNAPSHOT